### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Inspect][MAS4.2.1][Eclipse][Wipro]:Name property is empty for edit in "Run on web app for containers dialog"

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1402]
**Repro Steps:**

1. Open and Start Eclipse.
2. Create a new dynamic web project.
3. Navigate to and select Project ->Azure->Publish to web app for containers
4. "Run on web app for containers" dialog will open. 
5. In "Run on web app for containers" dialog, tab to and select  "Azure container registry" button.
6. Tab to Docker File edit  and observe the name property using inspect.

**Actual Result:**
Name property is empty for the edit box

**Narrator behaviour:**
Tab
Editing

**NVDA behaviour:**
edit
blank

**Expected Result:**
Name property should be simple and meaningful.
Name property for the edit should be "Docker File".

**Platform:**
Eclipse Version:
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
OSBuild Version: Version 1709(Os Build 16299.192)
![run on web app for containers-edit-no name](https://user-images.githubusercontent.com/22956130/35684788-f970092c-078d-11e8-99cc-cfcb6f517658.jpg)
![run on web app for containers-edit-no name-narrator](https://user-images.githubusercontent.com/22956130/35684794-fcb3d35c-078d-11e8-8b86-f80bdb602c22.jpg)
![run on web app for containers-edit-no name-nvda](https://user-images.githubusercontent.com/22956130/35684799-ff2cc508-078d-11e8-8f8d-ed58b3b2aae8.jpg)

**Comment by Eskibear** [comment]
already fixed in #1590

**Comment by Eskibear** [comment]
shipped in v3.6.0

**Comment by WiproPSAccTest** [comment]
Verified the issue in V.3.6.0 .Issue got fixed.
Please add closed tag.
![1402-fixed](https://user-images.githubusercontent.com/22956130/39761180-085bb73a-52f5-11e8-9c09-c2aeb11d974b.jpg)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749835
